Deployment/Kombit.Samples.CH.WebsiteDemo/Web.config

```diff
@@ -1,61 +1,63 @@
   </appSettings>
   <log4net>
     <root>
-      <level value="All" />
-      <appender-ref ref="LogFileAppender" />
+      <level value="All"/>
+      <appender-ref ref="LogFileAppender"/>
     </root>
     <appender name="LogFileAppender" type="log4net.Appender.RollingFileAppender">
-      <param name="File" value="log.txt" />
-      <param name="AppendToFile" value="true" />
-      <rollingStyle value="Size" />
-      <maxSizeRollBackups value="10" />
-      <maximumFileSize value="10MB" />
-      <staticLogFileName value="true" />
+      <param name="File" value="log.txt"/>
+      <param name="AppendToFile" value="true"/>
+      <rollingStyle value="Size"/>
+      <maxSizeRollBackups value="10"/>
+      <maximumFileSize value="10MB"/>
+      <staticLogFileName value="true"/>
       <layout type="log4net.Layout.PatternLayout">
-        <param name="ConversionPattern" value="%-5p%d{yyyy-MM-dd HH:mm:ss}-%m%n" />
+        <param name="ConversionPattern" value="%-5p%d{yyyy-MM-dd HH:mm:ss}-%m%n"/>
       </layout>
     </appender>
   </log4net>
   <location path="MyPage.aspx">
     <system.web>
       <authorization>
-        <deny users="?" />
+        <deny users="?"/>
       </authorization>
     </system.web>
   </location>
   <Federation auditLoggingType="dk.nita.saml20.Logging.Log4NetAuditLogger, dk.nita.saml20.ext.audit.log4net" xmlns="urn:dk.nita.saml20.configuration">
     <SessionTimeout>30</SessionTimeout>
-    <SigningCertificate findValue="4678237245FEDC8059D113675955DFB870D36BF4" storeLocation="LocalMachine" storeName="My" x509FindType="FindByThumbprint" />
+        <SigningCertificates>
+      <SigningCertificate findValue="21ad7d2d4280765bfe113b7dd5d62736c34e37bd" storeLocation="LocalMachine" storeName="My" x509FindType="FindByThumbprint"/>
+    </SigningCertificates>
     <AllowedAudienceUris>
-      <Audience>https://saml.claimapp.adgangsstyring.eksterntest-stoettesystemerne.dk</Audience>
+      <Audience>https://spdemo.gcm.safewhere.com</Audience>
     </AllowedAudienceUris>
     <Actions>
-      <clear />
-      <add name="SetSamlPrincipal" type="dk.nita.saml20.Actions.SamlPrincipalAction, dk.nita.saml20" />
-      <add name="LogAction" type="Kombit.Samples.CH.WebsiteDemo.LogAction, Kombit.Samples.CH.WebsiteDemo" />
-      <add name="Redirect" type="dk.nita.saml20.Actions.RedirectAction, dk.nita.saml20" />
+      <clear/>
+      <add name="SetSamlPrincipal" type="dk.nita.saml20.Actions.SamlPrincipalAction, dk.nita.saml20"/>
+      <add name="LogAction" type="Kombit.Samples.CH.WebsiteDemo.LogAction, Kombit.Samples.CH.WebsiteDemo"/>
+      <add name="Redirect" type="dk.nita.saml20.Actions.RedirectAction, dk.nita.saml20"/>
     </Actions>
   </Federation>
   <SAML20Federation xmlns="urn:dk.nita.saml20.configuration">
     <ShowError>true</ShowError>
-    <ServiceProvider id="https://saml.claimapp.adgangsstyring.eksterntest-stoettesystemerne.dk" server="https://claimapp.adgangsstyring.eksterntest-stoettesystemerne.dk">
```

Comment:
Should keep the old url (even though we will update it with a new url later)

Reply:
reverted to the old url